docs/DafnyRef/UserGuide.md

```diff
@@ -1441,6 +1466,21 @@ These options instruct Dafny to print various information about your
 program during processing, including variations of the original source
 code (which can be helpful for debugging).
 
+* `--use-basename-for-filename` - when enabled, just the filename without the 
+directory path is used in error messages; this make error message shorter and 
+not tied to the local environment (which is a help in testing)
+
+* `--output`, `-o` - location of output files [translate, build]
+
+* `--show-snippets` - include with an error message some of the source code text
+in the neighborhood of the error; the error location (file, line, column) is always given
+
+* `--solver-log <file>` - [verification only] the file in which to place the SMT text sent to the solver
+
+* `--log-format <configuration>` - [verification only] logs information about verification performance
```

Comment:
Can we move the explanation of the various permitted configurations / file formats here (from the legacy options section)?

Reply:
I'll change this (as I did for other options) in a next PR.